docs/CEAP_en_US.md

```diff
@@ -0,0 +1,100 @@
+Article 1 - It can use, above the calculated quota for it's State, an extra of:
+1. BRL 1353,04, the Congress Person who's:
+  * Leader of a political party, parliament block, Minority, or leader of the Government in the Chamber of Deputies or the National Congress.
```

Comment:
According to this source (http://www.camara.leg.br/Internet/Deputado/bancada.asp), the Minority is addressed as a political party and it uses capital M.